Mathlib/CategoryTheory/Products/Basic.lean

```diff
@@ -254,50 +248,30 @@ section
 variable (C)
 
 /-- The diagonal functor. -/
+@[simps! obj obj_fst obj_snd map map_fst map_snd]
```

Comment:
If `obj` is a simp lemma, I would doubt that `obj_fst` and `obj_snd` are necessary. Similarly for `map`.